client/src/actions/getMostFavoriteRecipes.js

```diff
@@ -0,0 +1,20 @@
+import axios from 'axios';
+import { GET_MOST_FAVORITED_RECIPES } from './types';
+
+const getMostFavoritesSuccess = favorites => ({
+  type: GET_MOST_FAVORITED_RECIPES,
+  favorites
+});
+
+
+const getMostFavorites = () => dispatch => axios
+  .get('/api/v1/recipes/most-favorites')
+  .then((response) => {
+    console.log('>>>>getMEssg', response.data);
+    dispatch(getMostFavoritesSuccess(response.data));
+  })
+  .catch((err) => {
+    console.log(err);
```

Comment:
### Nitpick

You don't want to always log your errors, you'll want to do something with it.